demo/services/availableComponentsService.ts

```diff
@@ -94,11 +94,11 @@ export default class AvailableComponentsService {
           '/tree-view',
           require('./../views/tree-view/basic.html'), 'demoTreeView as vm')
       ]),
-      new AvailableGroup('miq-selector-tree', 'Tree selector component', '/selector-tree', [
-        new AvailableComponent('miq-selector-tree',
+      new AvailableGroup('tree-selector', 'Tree selector component', '/tree-selector', [
```

Comment:
Let's use the same group as the `tree-view`, you'll need to rename/move the `basic.html` but those two components belong to a single group.